src/Caching/Cache.php

```diff
@@ -98,6 +98,45 @@ public function load($key, $fallback = NULL)
 
 
 	/**
+	 * Reads multiple items from the cache
+	 * @param array
+	 * @param callable
+	 * @return array
+	 */
+	public function bulkLoad(array $keys, $fallback = NULL)
```

Comment:
What if you want e.g. three values whose value are calculated in a different way? You would have to build this logic by if/switch branching inside the callback. That feels a bit ugly...